Fakebook.Posts/Fakebook.Posts.UnitTests/PostTest.cs

```diff
@@ -0,0 +1,134 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Fakebook.Posts.Domain.Models;
+using Xunit;
+
+namespace Fakebook.Posts.UnitTests
+{
+    public class PostTests
+    {
+        private const int Id = 1;
+
+        [Fact]
+        public void Post_Constructor_Pass()
+        {
+            //arrange
+            const string Content = "This is some content";
+            const string userEmail = "damion.silver@test.com";
+            //act
+            Post newpost = new Post(userEmail, Content);
+        }
+
+        [Fact]
+        public void Post_EmailIsNotNull()
```

Comment:
like i said this test isn't doing anything for us, given Post_EmailEquality, so just take it out.
Post_EmailisFormattedCorrectly is also not doing anything Post_EmailEquality is not also testing.

if a method returns "abc", we don't need three unit tests to test each character one by one, one unit test will do.

Reply:
I deleted this method.

Reply:
read the comment carefully, i'm also talking about `Post_EmailisFormattedCorrectly`

Reply:
I removed the formatted email method Post_EmailisFormattedCorrectly and Comment_EmailisFormattedCorrectly method.